scripts/Inelastic/Direct/ReductionWrapper.py

```diff
@@ -448,6 +448,13 @@ def reduce(self,input_file=None,output_directory=None):
         if output_directory:
             config['defaultsave.directory'] = str(output_directory)
 
+        #propman = self.reducer.prop_man
+        #if hasattr(propman,'fix_file_extension'):
+        #    fix_file_extension = propman.fix_file_extension
+        #else:
+        #    fix_file_extension = False
```

Comment:
If the commented out code is not needed then just remove it.

Reply:
There is some misagreement between IS on MARI and others. I modified code slightly and put additional comments to explain that.